### PR TITLE
fix: keybord overlays reaction picker WPB-3090

### PR DESCRIPTION
**Pull request by MarcinR**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-3090" title="WPB-3090" target="_blank">BugWPB-3090</a>  [iOS] Reactions: When long tapping on a message, selecting the custom emoji option, and looking for specific characters, the keyboard overlays the emoji selection
  </td></table>
  <br />
 

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability

----

# What's new in this PR?
If user open reaction picker and start typing in search bar then keyboard overlaps emojis at the botom. To prevent that Im increasing bottom inset of collection view so last row of cells is above keyboard.

### CODE REVIEW

File: .ruby-version
Comment (agisilaos) [inline .ruby-version:1]: Why did you commit the ruby version? I don't think it's needed. cc: @netbe

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.2.0
```
Comment (KaterinaWire) [inline .ruby-version]: Why do we need this change?

**Comment by KaterinaWire** [comment]
@MarcinR Could you add a description to this PR? I see an offset for the keyboard. Have you tested it for iPad as well?

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/MessageActions/CompleteReactionPicker/ReactionsCollectionView.swift [hunk 2]
```diff
@@ -28,18 +28,36 @@ final class ReactionsCollectionView: UICollectionView {
...
         layout.scrollDirection = .vertical
         layout.minimumLineSpacing = 0
         layout.minimumInteritemSpacing = 0
     }
 
-    @available(*, unavailable)
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    func configureObservers() {
+        NotificationCenter.default.addObserver(self, selector: #selector(handleKeyboardPresentation), name: UIResponder.keyboardWillShowNotification, object: nil)
```
Comment (KaterinaWire) [inline wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/MessageActions/CompleteReactionPicker/ReactionsCollectionView.swift:51]: Did you check `KeyboardBlockObserver` ? Can we reuse it?

Comment (MarcinR) [reply wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/MessageActions/CompleteReactionPicker/ReactionsCollectionView.swift:51]: Yes, I've seen it. But it requires more code and make code more complicated and harder to read

**Comment by MarcinR** [comment]
> @MarcinR Could you add a description to this PR? I see an offset for the keyboard. Have you tested it for iPad as well?

**Review by agisilaos — APPROVED** [review]

**Comment by agisilaos** [comment]
@MarcinR Works great on both iPhone and iPad!

**Pull request merged**